rapids-cmake/cython-core/create_modules.cmake

```
@@ -0,0 +1,158 @@
+# =============================================================================
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+include_guard(GLOBAL)
+
+#[=======================================================================[.rst:
+rapids_cython_create_modules
+----------------------------
+
+.. versionadded:: v24.02.00
+
+Generate C(++) from Cython and create Python modules.
+
+.. code-block:: cmake
+
+  rapids_cython_create_modules([CXX] [SOURCE_FILES <src1> <src2> ...] [LINKED_LIBRARIES <lib1> <lib2> ... ]  [INSTALL_DIR <install_path>] [MODULE_PREFIX <module_prefix>] )
```

Should this be nicely wrapped [like other example docstrings](https://github.com/rapidsai/rapids-cmake/blob/2969d42a155d6045a17cc7e79c83783befd3b7d5/rapids-cmake/cpm/gbench.cmake#L32)? Also there's some inconsistent spacing.
```suggestion
  rapids_cython_create_modules([CXX]
                               [SOURCE_FILES <src1> <src2> ...]
                               [LINKED_LIBRARIES <lib1> <lib2> ...]
                               [INSTALL_DIR <install_path>]
                               [MODULE_PREFIX <module_prefix>])
```